sdk/eval.go

```diff
@@ -1,11 +1,19 @@
 package sdk
 
+import (
+	"time"
+
+	"github.com/hashicorp/nomad-autoscaler/helper/uuid"
```

Comment:
I would expect this to be caught by the `check-sdk` make target which disallows us from importing internal packages into the `sdk` package.